### DISCUSSION TITLE
Heat Release Rate (HRR) for a Fire Pit

### DISCUSSION THREAD

**Discussion by aking36** [firemodels/cfast#2133] [Q&A]
Can anyone point me to some specific information on how to determine the HHR time dependent information needed to run a simulation for a traditional back yard fire pit scenario? I know that we burned 7.0 kg of hardwood logs which have a heat of combustion of 14466kJ/kg, but I have no idea how to construct the HRR graph on the fires tab. For the fires tab, it appears I need to know the HRR (kW) at different points in time. How does someone figure this out or at minimum approximate it?

**Comment by mcgratta** [comment]
Suppose you burned 7 kg of hardwood chips, and then you burned a single 7 kg log. Would you expect the same burning rate? Of course not. So depending on how you chop the wood, and depending on how much moisture is in the wood, and how "green" the wood is, matters. Thus, no one has a HRR curve just for you. You would need to burn the wood under a calorimeter and measure its HRR.
